R-package/R/lgb.Predictor.R

```diff
@@ -1,13 +1,15 @@
 #' @importFrom methods is new
-#' @importClassesFrom Matrix dsparseMatrix dsparseVector dgCMatrix dgRMatrix
 #' @importFrom R6 R6Class
 #' @importFrom utils read.delim
+#' @importClassesFrom Matrix dsparseMatrix dsparseVector dgCMatrix dgRMatrix CsparseMatrix RsparseMatrix
 Predictor <- R6::R6Class(
 
   classname = "lgb.Predictor",
   cloneable = FALSE,
   public = list(
 
+    fast_predict_config = list(),
```

Comment:
Similar to my other comment about the `Booster` class...could you please make this a private attribute, or explain why it's necessary for it to be public? Making this private would give us greater flexibility to change this implementation in the future without breaking user code.

Reply:
Moved.